FFXCS.asl

```diff
@@ -679,6 +680,7 @@ update
     {
         print("Luca - Wakka is injured");
         game.WriteValue(modules.First().BaseAddress+0xD2D67C, 518); //CutsceneID
+        game.WriteValue(modules.First().BaseAddress+0xD2CDE5, (byte)11); // Special flag to indicate when scene has been played
```

Comment:
This is not needed for the skip, but not sure if the game will get in a weird state without it, so setting just in case